### PR TITLE
FAD-5695 fixed library to work with ioredis client

### PR DESCRIPTION
**Pull request by gwalls**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "redis-timeseries",
+  "name": "@sparkpost/redis-timeseries",
   "version": "0.4.0",
```
Comment (beardyman) [inline package.json]: make 1.0.0

File: src/timeseries.js [hunk 2]
```diff
@@ -113,7 +113,15 @@ TimeSeries.prototype.getHits = function(key, gran, count, callback) {
     }
 
     for(var ts=from, i=0, data=[]; ts<=to; ts+=properties.duration, i+=1) {
-      data.push([ts, results[i] ? parseInt(results[i], 10) : 0]);
+      var result = results[i];
+      if (result instanceof Array) {
```
Comment (beardyman) [inline src/timeseries.js]: I would comment here explaining that different clients libs return different things ... just explain why you're doing what you're doing.

**Review by beardyman — CHANGES_REQUESTED** [review]
What about tests?  We should update/add to whatever tests are in place and probably set this up in CI to have them auto run.

**Comment by gwalls** [comment]
@beardyman there are currently no tests and no CI. I can set these up if you think it's worth the time right now?

**Comment by beardyman** [comment]
@gwalls Theres a tests folder with a simple test in it.  I guess we should probably wait and defer tests since it could be a decent undertaking to write tests for this stuff.  I'm going to ping a few peeps about it and see how they feel.

**Comment by beardyman** [comment]
deferring tests from convo irl

**Review by beardyman — APPROVED** [review]

**Pull request merged**